exe/wallet/jormungandr/Main.hs

```diff
@@ -341,7 +336,10 @@ cmdServe = command "serve" $ info (helper <*> cmd) $ mempty
                     handleGenesisNotFound (sb, tracer)
                 Left (ErrGetBlockchainParamsIncompleteParams _) ->
                     handleNoInitialPolicy tracer
-            return (nl, genesis)
+
+            let nl' = coerceBlock <$> nl
+            let tl = Jormungandr.newTransactionLayer @n block0H
+            Wallet.newWalletLayer tracer blockchainParams db nl' tl
```

Comment:
What's the point of making a `rawNetworkLayer`, and only use `nl' = coerceBlock <$> nl` ? Am I missing something or could this be simply using `mkNetworkLayer` ?

Reply:
> Am I missing something or could this be simply using `mkNetworkLayer`

Correct. Currently there is no point. But it is here I'd like to use a different `coerceBlock <$> nl` for `StakePool.Metrics`.